getTime.tsx

```diff
@@ -0,0 +1,21 @@
+export const getCurrentTime = () => {
+  const date = new Date();
+  const day = date.getUTCDate();
+  const month = date.getUTCMonth() + 1;
+  const year = date.getUTCFullYear();
+  const dayMonthYear = `${day}-${month}-${year}`;
```

Comment:
Check this info to format it

https://stackoverflow.com/questions/3552461/how-do-i-format-a-date-in-javascript